### ISSUE TITLE
Fix type matching for signatures

### ISSUE THREAD

**Issue by XVilka** [rizinorg/rizin#1300]
### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             | -
| File format of the file you reverse (mandatory)      | ELF
| Architecture/bits of the file (mandatory)            | x86/64
| `rizin -v` full output, **not truncated** (mandatory)  | 

### Expected behavior

Successful rename of the function arguments

### Actual behavior

```
[XX] db/cmd/cmd_zignature add types and function name on sig match
RZ_NOPLUGINS=1 rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'zs *
za main o 0x00004070
za main t func.sym.new_function_name.ret="int" func.sym.new_function_name.args=2 func.sym.new_function_name.arg.0="\"int,NEWARGC\"" func.sym.new_function_name.arg.1="\"char **,NEWARGV\""
za main n sym.new_function_name
s main
af
e zign.match.offset=true
z.
afs
afn
' bins/elf/ls
-- stdout
--- expected
+++ actual
@@ -1,2 +1,2 @@
-int sym.new_function_name(int NEWARGC, char ** NEWARGV);
+void sym.new_function_name(int argc, char ** argv);
 sym.new_function_name
```

**Comment by ret2libc** [comment]
@XVilka is this still a problem?

**Comment by XVilka** [comment]
Yes, it's more of the zignatures problem rather than types themselves

**Comment by XVilka** [comment]
Will be solved by removing old signatures after the 0.4.0 release.
